samples/SwiftDemoApp/SwiftDemoApp/Samples.swift

```diff
@@ -13,25 +13,40 @@
  * limitations under the License.
  */
 
-import Foundation
+import UIKit
 
-class Samples {
-  class func loadSamples() -> [Any] {
-    return [ newDemo(controller: ClusteringViewController.self,
-                     title: "Clustering",
-                     description: "Marker Clustering"),
-             newDemo(controller: KMLViewController.self,
-                     title: "KML",
-                     description: "KML Rendering"),
-             newDemo(controller: GeoJSONViewController.self,
-                     title: "GeoJSON",
-                     description: "GeoJSON Rendering"),
-             newDemo(controller: HeatmapViewController.self,
-                     title: "Heatmaps",
-                     description: "Heatmaps")]
-  }
+enum Samples: CaseIterable{
```

Comment:
```suggestion
enum Samples: CaseIterable {
```

Reply:
Since this is now an enum, I think this should also be renamed to singular form "`Sample`"

Reply:
done 👍